pkg/chart/loader/load.go

```diff
@@ -205,3 +206,48 @@ func LoadFiles(files []*BufferedFile) (*chart.Chart, error) {
 
 	return c, nil
 }
+
+// LoadValues loads chat values from a reader.
```

Comment:
typo

And values are not necessary from a chart. They can be either be a chart's default values. Or user supplied values

Reply:
thanks, updated here